eppo_core/src/event_ingestion/event.rs

```diff
@@ -1,11 +1,18 @@
-use crate::timestamp::Timestamp;
 use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+
+use crate::timestamp::Timestamp;
 
+#[serde_as]
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Hash)]
 pub(super) struct Event {
     pub uuid: uuid::Uuid,
+
+    #[serde(with = "chrono::serde::ts_milliseconds")]
     pub timestamp: Timestamp,
+
     #[serde(rename = "type")]
     pub event_type: String,
+
     pub payload: serde_json::Value,
-}
+}
```

Comment:
nit: please keep newlines at the end of files. Otherwise, git is detecting the last line as changes if we add new code after it (which results in dirty git history, which may hurt some git archeology in the future)

Reply:
was unintentional 👍🏽